frontend/src/components/SignForms/SignInForm/SignIn.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import Button from '@material-ui/core/Button';
+
+import EmailField from './SignComponents/EmailField';
+import PasswordField from './SignComponents/PasswordField';
+import SignLabel from './SignComponents/SignLabel';
+import { SignInFormik } from '../SignInterfaces';
```

Comment:
Remove this file and change the structure of SignIn page

Reply:
Done